public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.js

```diff
@@ -667,9 +667,12 @@ class LibreTextsGlossarizer {
     }
 }
 
+//enable debug mode
+LibreTexts.debug.glossarizer = true;
 
 //Self Initialize
 let libretextGlossary = new LibreTextsGlossarizer(); // Needs to be accessible to the sidebar buttons
-window.addEventListener('load', ()=>{
-    libretextGlossary.makeGlossary();
+window.addEventListener('load', () => {
```

Comment:
Add check maybe for document.currentScript src tag `includes('files.libretexts.org')`